app/components/agenda-manager/edit.js

```diff
@@ -5,15 +5,33 @@ import { inject as service } from '@ember/service';
 
 export default class AgendaManagerEditComponent extends Component {
   @service documentService;
-
+  oldGeplandOpenbaar;
+  constructor() {
+    super(...arguments);
+    this.oldGeplandOpenbaar = this.args.itemToEdit.geplandOpenbaar;
```

Comment:
why are you not just using a getter?

Reply:
Because I want to get the original value when the component was created not the actual value of the variable